src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -464,8 +464,13 @@ export class HostsComponent extends ListWithDetails implements OnDestroy, OnInit
   transformHostsData() {
     if (this.checkHostsFactsAvailable()) {
       _.forEach(this.hosts, (hostKey) => {
-        hostKey['memory_total_bytes'] = hostKey['memory_total_kb'] * 1024;
-        hostKey['raw_capacity'] = hostKey['hdd_capacity_bytes'] + hostKey['flash_capacity_bytes'];
+        hostKey['memory_total_bytes'] = !isNaN(hostKey['memory_total_kb'])
+          ? hostKey['memory_total_kb'] * 1024
+          : 'N/A';
+        hostKey['raw_capacity'] =
+          !isNaN(hostKey['hdd_capacity_bytes']) && !isNaN(hostKey['flash_capacity_bytes'])
+            ? hostKey['hdd_capacity_bytes'] + hostKey['flash_capacity_bytes']
+            : 'N/A';
```

Comment:
honestly, if a ternary assignment is not a one liner I would prefer a usual if statement.

Reply:
I avoided writing if else block twice)